.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.idea
```

Comment:
Perhaps you should set that in gitconfig's `core.excludesfile` or `~/.gitignore`